### ISSUE TITLE
Can't disable warnings for Opera Mini

### ISSUE THREAD

**Issue by websolutions-hamburg** [ismay/stylelint-no-unsupported-browser-features#30]
Hello.

I use this browser configuration`"browsers": ["> 1%", "Last 2 versions", "not OperaMin >= 0"]` but I get still warnings for Opera Mini like this one:

```
Unexpected browser feature "background-img-opts" is only supported by Opera Mini all (plugin/no-unsopported-browser-features)
```
for this css rule

```css
background-size: cover;
```

How can I disable this warnings? Thought "not OperaMin >= 0" will help, but it doesn't work.

**Comment by ismay** [comment]
Hi @websolutions-hamburg, check the [browserslist docs](https://github.com/ai/browserslist) for further information on specific browser queries. That's what this library uses as well.

Maybe this example helps as well: https://github.com/ismay/ismaywolff.nl/blob/develop/.browserslistrc

**Comment by websolutions-hamburg** [comment]
> Maybe this example helps as well: https://github.com/ismay/ismaywolff.nl/blob/develop/.browserslistrc

Thanks allot!
I searched a long time, but I never found the `all` option. It is not documented on the browserlist docs.

This works as expected: `not OperaMini all`
